datahub-web-react/src/app/entity/glossaryTerm/profile/GlossaryTermHeader.tsx

```diff
@@ -0,0 +1,63 @@
+import { Avatar, Divider, Space, Tooltip, Typography } from 'antd';
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { EntityType } from '../../../../types.generated';
+import { useEntityRegistry } from '../../../useEntityRegistry';
+import defaultAvatar from '../../../../images/default_avatar.png';
+
+type Props = {
+    definition: string;
+    sourceRef: string;
+    sourceUrl: string;
+    ownership?: any;
+    customProperties: Array<{ key: string; value: string }>;
+};
+export default function GlossaryTermHeader({ definition, sourceRef, sourceUrl, ownership, customProperties }: Props) {
+    const entityRegistry = useEntityRegistry();
+    return (
+        <>
+            <Space direction="vertical" size="middle">
+                <Typography.Paragraph>{definition}</Typography.Paragraph>
+                <Space split={<Divider type="vertical" />}>
+                    <Typography.Text>Source</Typography.Text>
+                    <Typography.Text strong>{sourceRef}</Typography.Text>
+                    {sourceUrl && (
+                        <Link to={sourceUrl} target="_blank">
+                            view source
+                        </Link>
+                    )}
+                </Space>
+                {customProperties &&
+                    customProperties.map((data) => {
+                        return (
+                            <Space split={<Divider type="vertical" />}>
+                                <Typography.Text>{data.key}</Typography.Text>
+                                <Typography.Text strong>{data.value}</Typography.Text>
+                            </Space>
+                        );
+                    })}
+
```

Comment:
We usually show the custom properties as a table below- would it make sense to have a properties and ownership tab?